python/lsst/ts/externalscripts/auxtel/latiss_base_align.py

```diff
@@ -0,0 +1,788 @@
+# This file is part of ts_externalcripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["LatissBaseAlign"]
+
+import abc
+import time
+import types
+import yaml
+import typing
+import dataclasses
+
+import numpy as np
+
+from lsst.ts import salobj
+from lsst.ts.observatory.control.utils import RotType
+from lsst.ts.observatory.control.auxtel.atcs import ATCS
+from lsst.ts.observatory.control.auxtel.latiss import LATISS
+from lsst.ts.idl.enums.ATPtg import WrapStrategy
+
+STD_TIMEOUT = 10
+
+
+@dataclasses.dataclass
+class LatissAlignResults:
+    zernikes: typing.Tuple[float, float, float]
+    zernikes_rot: typing.Tuple[float, float, float]
+    offset_hex: typing.Tuple[float, float, float]
+    offset_tel: typing.Tuple[float, float, float]
+
+
+class LatissBaseAlign(salobj.BaseScript, metaclass=abc.ABCMeta):
+    """Implements generic behavior for scripts that execute curvature wavefront
+    sensing, abstracting the part that performs the measurements.
+
+    This meta Script implements the basic functionality of finding targets and
+    taking the (intra/extra focal) data. Child classes are left to implement
```

Comment:
As a generic comment, should we not be using semantic line versioning for all comments/docstrings etc?

Reply:
Uhm, not sure I understand what you mean. Can you rephrase the question?

Reply:
uhm.. I'm not sure that would work for docstrings though... Because we have limits on the line lengths and such.

Reply:
Not worth holding this up, but shouldn't it be done anyways to the extent possible? Meaning a new sentence goes on a new line? It should reduce the amount of differences when making text changes and even help avoid passing limits.